sql/core/src/test/scala/org/apache/spark/sql/GeneratorFunctionSuite.scala

```diff
@@ -219,20 +219,21 @@ class GeneratorFunctionSuite extends QueryTest with SharedSparkSession {
 
   test("inline raises exception on array of null type") {
     val m = intercept[AnalysisException] {
-      spark.range(2).selectExpr("inline(array())")
```

Comment:
can we leave existing UTs alone and add extra tests in this file?

Reply:
I was trying to update things to match how the explode's were being tested, which is mostly scala functions. I can duplicate some of them with the scala function instead if that would be preferred